doc/contributing/reviewing-contributions.xml

```diff
@@ -516,8 +516,16 @@
   </para>
 
   <para>
-   TODO: add the procedure to request merging rights.
+   To be granted this level of access you should:
   </para>
+  <orderedlist>
+   <listitem>
+    <para>Be active and well-known in the community</para>
```

Comment:
Note that these are not officially accepted terms -- those are just what I, personally, would use to decide.

Reply:
Fair. "well-known" isn't exactly quantifiable but it read better than that TODO...